django/contrib/admin/templates/registration/password_reset_form.html

```diff
@@ -20,7 +20,9 @@
         <label for="id_email">{% translate 'Email address:' %}</label>
         {{ form.email }}
     </div>
-    <input type="submit" value="{% translate 'Reset my password' %}">
+    <div class="submit-row">
+         <input type="submit" value="{% translate 'Reset my password' %}">
+    </div>
```

Comment:
To get a similar view we also should move `input` outside of `fieldset` and add `flex-container`:
```diff
diff --git a/django/contrib/admin/templates/registration/password_reset_confirm.html b/django/contrib/admin/templates/registration/password_reset_confirm.html
index 11c8500e6d..a07387f4bf 100644
--- a/django/contrib/admin/templates/registration/password_reset_confirm.html
+++ b/django/contrib/admin/templates/registration/password_reset_confirm.html
@@ -16,22 +16,26 @@
 <p>{% translate "Please enter your new password twice so we can verify you typed it in correctly." %}</p>
 
 <form method="post">{% csrf_token %}
-<fieldset class="module aligned">
+  <fieldset class="module aligned">
     <input class="hidden" autocomplete="username" value="{{ form.user.get_username }}">
     <div class="form-row field-password1">
-        {{ form.new_password1.errors }}
+      {{ form.new_password1.errors }}
+      <div class="flex-container">
         <label for="id_new_password1">{% translate 'New password:' %}</label>
         {{ form.new_password1 }}
+      </div>
     </div>
     <div class="form-row field-password2">
-        {{ form.new_password2.errors }}
+      {{ form.new_password2.errors }}
+      <div class="flex-container">
         <label for="id_new_password2">{% translate 'Confirm password:' %}</label>
         {{ form.new_password2 }}
+      </div>
     </div>
-    <div class="submit-row">
-         <input type="submit" value="{% translate 'Change my password' %}">
-    </div>
-</fieldset>
+  </fieldset>
+  <div class="submit-row">
+     <input type="submit" value="{% translate 'Change my password' %}">
+  </div>
 </form>
 
 {% else %}
diff --git a/django/contrib/admin/templates/registration/password_reset_form.html b/django/contrib/admin/templates/registration/password_reset_form.html
index 104c61fc54..512cab57f1 100644
--- a/django/contrib/admin/templates/registration/password_reset_form.html
+++ b/django/contrib/admin/templates/registration/password_reset_form.html
@@ -14,16 +14,18 @@
 <p>{% translate 'Forgotten your password? Enter your email address below, and we’ll email instructions for setting a new one.' %}</p>
 
 <form method="post">{% csrf_token %}
-<fieldset class="module aligned">
+  <fieldset class="module aligned">
     <div class="form-row field-email">
-        {{ form.email.errors }}
+      {{ form.email.errors }}
+      <div class="flex-container">
         <label for="id_email">{% translate 'Email address:' %}</label>
         {{ form.email }}
+      </div>
     </div>
-    <div class="submit-row">
-         <input type="submit" value="{% translate 'Reset my password' %}">
-    </div>
-</fieldset>
+  </fieldset>
+  <div class="submit-row">
+     <input type="submit" value="{% translate 'Reset my password' %}">
+  </div>
 </form>
 
 {% endblock %}

```

----


![image](https://github.com/django/django/assets/2865885/9fb1e611-104d-41f6-bf50-ed8bf867aa0d)

-----

![image](https://github.com/django/django/assets/2865885/f315bbd3-5503-426e-ac02-c5f2fcc8ef68)